php-practice_1.php

```diff
@@ -1,41 +1,137 @@
 <?php
 // Q1 変数と文字列
-
+$name = '「奈良」';
+echo '私の名前は'.$name.'です。';
```

Comment:
[must]
`.`前後はスペースを設けてください！
```suggestion
echo '私の名前は' . $name . 'です。';
```

Reply:
申し訳ございません、修正いたしました。

Reply:
[imo]
謝るよりは指摘してもらったことに対して感謝するほうがポジティブなコミュニケーションになると思います！